app/services/idv/actions/in_person/cancel_update_address_action.rb

```diff
@@ -2,12 +2,14 @@ module Idv
   module Actions
     module InPerson
       class CancelUpdateAddressAction < Idv::Steps::DocAuthBaseStep
+        include Idv::Steps::TempMaybeRedirectToVerifyInfoHelper
```

Comment:
```suggestion
        include Idv::Steps::TempMaybeRedirectToVerifyInfoHelper

```
It's a nit, but I'd add a space below the include here and in `CancelUpdateSsnAction` and `CancelUpdateStateIdAction` as well.